lib/src/sse_proxy_handler.dart

```diff
@@ -34,14 +34,14 @@ String _sseHeaders(String origin) => 'HTTP/1.1 200 OK\r\n'
 /// simply forwards data back and forth between clients and the actual server.
 class SseProxyHandler {
   final _httpClient = http.Client();
-  shelf.Handler _incomingMessageProxyHandler;
-  final String _proxyName;
+  shelf.Handler? _incomingMessageProxyHandler;
```

Comment:
#nit we could probably make this `late` instead of nullable

Reply:
done